habitat/tasks/nav/nav_task.py

```diff
@@ -719,6 +720,30 @@ def __init__(
         self.sensor_suite = SensorSuite(task_sensors)
         super().__init__(config=task_config, sim=sim, dataset=dataset)
 
+    @registry.register_task_action(name="move_forward")
+    def move_forward(self):
+        return self._sim.step(SimulatorActions.MOVE_FORWARD)
+
+    @registry.register_task_action(name="turn_left")
+    def turn_left(self):
+        return self._sim.step(SimulatorActions.TURN_LEFT)
+
+    @registry.register_task_action(name="turn_right")
+    def turn_right(self):
+        return self._sim.step(SimulatorActions.TURN_RIGHT)
+
+    @registry.register_task_action(name="look_up")
+    def look_up(self):
+        return self._sim.step(SimulatorActions.LOOK_UP)
+
+    @registry.register_task_action(name="look_down")
+    def look_down(self):
+        return self._sim.step(SimulatorActions.LOOK_DOWN)
+
+    @registry.register_task_action(name="stop")
+    def stop(self):
+        return self._sim.step(SimulatorActions.STOP)
```

Comment:
We should probably rename `SimulatorActions` to `Actions` or `BaseActions` as `STOP` isn't a simulator action anymore :)

Reply:
@erikwijmans, but `SimulatorActions`is used only as bridge between Simulator Actions and Task Actions. There will be no task action enum anymore. Registry will be used for that. I can remove `STOP` action for HSIM here.